tests/includes/listener-loader.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Replaces our copy of `SpeedTrapListener` and includes WP Test lib SpeedTrapListener. Note that WP have a compatibility layer for it: http://develop.svn.wordpress.org/trunk/tests/phpunit/includes/listener-loader.php